### PR TITLE
[Feature] Dark Mode

### PR DESCRIPTION
**Pull request by octoper**
Add dark mode for Pest documentation

### CODE REVIEW

**Comment by owenvoke** [comment]
I'll have a look over this shortly!

**Comment by nunomaduro** [comment]
Gonna let @owenvoke and/or @AlexMartinFR check this one.

**Review by owenvoke — APPROVED** [review]
LGTM! Let @AlexMartinFR check too though.

@octoper, I just wondered, is the homepage meant to have a dark theme too? 👍🏻

**Comment by octoper** [comment]
@owenvoke I'm not sure if it's possible with the toggle based dark mode, only with `prefers-color-scheme` but I will check it out

**Comment by nunomaduro** [comment]
@octoper Can i see some screenshots?

**Comment by octoper** [comment]
@nunomaduro here you go (Need to find a White logo of AMP)
pestphp com test_

**Comment by nunomaduro** [comment]
What do you think about adding the dark mode just for the docs itself?

**Comment by octoper** [comment]
@nunomaduro Yeah this was my idea

File: public/mix-manifest.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-    "/js/app.js": "/js/app.js?id=b29f0c73dbe076a7f8c9",
-    "/css/app.css": "/css/app.css?id=72b1cba61c5ebdd056a5"
+    "/js/app.js": "/js/app.js",
```
Comment (AlexMartinFR) [inline public/mix-manifest.json:2]: The versioning would be needed for cache-busting purposes.

**Review by AlexMartinFR — COMMENTED** [review]
Everything looks OK.
I'll test it on my local machine tomorrow before approving 👍

**Comment by octoper** [comment]
Made some cool UI improvements @AlexMartinFR suggested and this PR is ready to get reviewed and merged!

**Review by owenvoke — APPROVED** [review]
Nice job fixing the accessibility issues with the search bar.

**Review by AlexMartinFR — APPROVED** [review]

**Comment by AlexMartinFR** [comment]
LGTM!

**Pull request merged**